content/implementation.md

```diff
@@ -6,17 +6,41 @@ order: 1
 
 EOF has been implemented in the following clients and projects.
 
-| |[3540](https://eips.ethereum.org/EIPS/eip-3540)|[3670](https://eips.ethereum.org/EIPS/eip-3670)|[4200](https://eips.ethereum.org/EIPS/eip-4200)|[4750](https://eips.ethereum.org/EIPS/eip-4750)|[5450](https://eips.ethereum.org/EIPS/eip-5450)|[6206](https://eips.ethereum.org/EIPS/eip-6206)|[7480](https://eips.ethereum.org/EIPS/eip-7480)|[7069](https://eips.ethereum.org/EIPS/eip-7069)|[7620](https://eips.ethereum.org/EIPS/eip-7620)|[7698](https://eips.ethereum.org/EIPS/eip-7698)|[663](https://eips.ethereum.org/EIPS/eip-663)|
-|--------------------------------------------------------------------------------------------|:---|:-----|:---|:---|:----|:---|:---|:---|:---|:---|:---|
-| [Megaspec](https://github.com/ipsilon/eof/blob/main/spec/eof.md)                           | ✅ | ✅   | ✅ | ✅ | ✅  | ✅ | ✅ | ✅ | ✅ | ✅ | ✅ |
-| EIP                                                                                        | ✅ | ✅   | ✅ | ✅ | ✅  | ✅ | ✅ | ✅ | ✅ | ✅ | ✅ |
-| Testing                                                                                    | ✅ | ✅   | ✅ | ✅ | ✅  | ✅ | 🚧 | 🚧 | 🚧 |    | ✅ |
-| [besu](https://github.com/hyperledger/besu/tree/mega-eof)                                  | ✅ | ✅   | ✅ | ✅ | ✅  | ✅ | ✅ | ✅ | ✅ | ✅ | ✅ |
-| erigon                                                                                     | 🚧 | 🚧   | 🚧 | 🚧 | 🚧  | 🚧 | 🚧 |    |    |    | 🚧 |
-| [ethereumjs](https://github.com/ethereumjs/ethereumjs-monorepo/pull/3440)                  | 🚧 | 🚧   | 🚧 | 🚧 | 🚧  | 🚧 | 🚧 | 🚧 | 🚧 | 🚧 | 🚧 |
-| [evmone](https://github.com/ethereum/evmone)                                               | ✅ | ✅   | ✅ | ✅ | ✅  | ✅ | ✅ | ✅ | ✅ | ✅ | ✅ |
-| [geth](https://github.com/ethereum/go-ethereum/pull/29518)                                 | 🚧 | 🚧   | 🚧 | 🚧 | 🚧  | 🚧 | 🚧 | 🚧 | 🚧 |    | 🚧 |
-| [nethermind](https://github.com/NethermindEth/nethermind/commits/feature/evm/eof)          | ✅ | ✅   | ✅ | ✅ | ✅  | ✅ | ✅ | ✅ | ✅ | ✅ | ✅ |
-| revm                                                                                       | ✅ | ✅   | ✅ | ✅ | ✅  | ✅ | ✅ | ✅ | ✅ | ✅ | ✅ |
-| [solidity (experimental)](https://github.com/ethereum/solidity/releases/tag/v0.8.29)       | ✅ | N/A  | ✅ | ✅ | N/A | ✅ | ✅ | ✅ | ✅ | ✅ | ✅ |
+|                | [eof-devnet-0]           | [eof-devnet-1]       | eof-devnet-2 |
+|----------------|--------------------------|----------------------|--------------|
```

Comment:
```suggestion
|----------------|:-------------------------|:---------------------|:-------------|
```

Align to the left:

![2025-04-28-052705_817x305_scrot](https://github.com/user-attachments/assets/78dcabc4-1b6a-4685-ac6f-f9389afc2251)